code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/ConsentExtension.java

```diff
@@ -331,45 +309,18 @@ private void shareCurrentConsents(final Event event) {
 		final Map<String, Object> xdmConsents = consentManager.getCurrentConsents().asXDMMap();
 
 		// set the shared state
-		ExtensionErrorCallback<ExtensionError> errorCallback = new ExtensionErrorCallback<ExtensionError>() {
-			@Override
-			public void error(final ExtensionError extensionError) {
-				MobileCore.log(
-					LoggingMode.DEBUG,
-					ConsentConstants.LOG_TAG,
-					String.format(
-						"ConsentExtension - Failed create XDM shared state. Error : %s.",
-						extensionError.getErrorName()
-					)
-				);
-			}
-		};
-
-		getApi().setXDMSharedEventState(xdmConsents, event, errorCallback);
+		getApi().createXDMSharedState(xdmConsents, event);
 
 		// create and dispatch an consent response event
 		final Event responseEvent = new Event.Builder(
 			ConsentConstants.EventNames.CONSENT_PREFERENCES_UPDATED,
-			ConsentConstants.EventType.CONSENT,
-			ConsentConstants.EventSource.RESPONSE_CONTENT
+			EventType.CONSENT,
+			EventSource.RESPONSE_CONTENT
```

Comment:
Should this event also include `.inResponseToEvent(event)` since this is a response event?